contracts/UniswapV3Staker.sol

```diff
@@ -88,13 +89,11 @@ contract UniswapV3Staker is
         require(params.rewardToken != address(0), 'INVALID_REWARD_ADDRESS');
         require(params.totalReward > 0, 'INVALID_REWARD_AMOUNT');
 
-        require(
```

Comment:
I removed the require as `safeTransferFrom` does a `require` check on the output.